lib/Graphs/PAG.cpp

```diff
@@ -50,6 +50,45 @@ const std::string PAGNode::toString() const {
     return rawstr.str();
 }
 
+/// Get shape and/or color of node for .dot display.
+const std::string PAGNode::getNodeAttributes() const {
```

Comment:
If this is only used in `DOTGraphTraits`. I would suggest putting only in that class.

Reply:
I'm not sure how to do that. Perhaps that can wait for later. There are several specializations of DOTGraphTraits and I'm not sure how to make one instantiation make use of another's instance.

Reply:
OK. I'm renaming is as you suggested (in my next push).